### ISSUE TITLE
Determine if any of the things we need to do with SLOBs are already covered by existing plugins

### ISSUE THREAD

**Issue by stephenmaturin** [jmfuch02/alss#5]
This is waiting on #4

**Comment by stephenmaturin** [comment]
Here is the GitHub repo for the web socket API: https://github.com/Palakis/obs-websocket
There are two available Python interfaces:
* https://github.com/KirillMysnik/obs-ws-rc
* https://github.com/IRLToolkit/simpleobsws

Neither looks particularly widely used.
